.gitignore

```diff
@@ -1,124 +1,331 @@
 # Crashlytics plugin (for Android Studio and IntelliJ)
 com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
+fabric.properties
 
+# Editor-based Rest Client
+.idea/httpRequests
 
-### IntelliJ Idea ###
-*.iml
-*.ipr
-*.iws
-.idea/
-/out/
+# Android studio 3.1+ serialized cache file
+.idea/caches/build_file_checksums.ser
 
-### Maven ###
+### Maven template
 target/
+pom.xml.tag
+pom.xml.releaseBackup
 pom.xml.versionsBackup
-#############
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+# https://github.com/takari/maven-wrapper#usage-without-binary-jar
+.mvn/wrapper/maven-wrapper.jar
 
-### Eclipse ###
-*.pydevproject
-.project
+### JDeveloper template
+# default application storage directory used by the IDE Performance Cache feature
+.data/
+
+# used for ADF styles caching
+temp/
+
+# default output directories
+classes/
+deploy/
+javadoc/
+
+# lock file, a part of Oracle Credential Store Framework
+cwallet.sso.lck
+### Eclipse template
 .metadata
-bin/**
-tmp/**
-tmp/**/*
+bin/
+tmp/
 *.tmp
 *.bak
 *.swp
 *~.nib
 local.properties
-.classpath
 .settings/
 .loadpath
+.recommenders
 
 # External tool builders
 .externalToolBuilders/
 
 # Locally stored "Eclipse launch configurations"
 *.launch
 
-# CDT-specific
+# PyDev specific (Python IDE for Eclipse)
+*.pydevproject
+
+# CDT-specific (C/C++ Development Tooling)
 .cproject
 
+# CDT- autotools
+.autotools
+
 # Java annotation processor (APT)
 .factorypath
 
+# PDT-specific (PHP Development Tools)
+.buildpath
+
 # sbteclipse plugin
 .target
 
+# Tern plugin
+.tern-project
+
 # TeXlipse plugin
 .texlipse
 
-# PDT-specific
-.buildpath
-###############
-
-### Java ###
-*.class
+# STS (Spring Tool Suite)
+.springBeans
 
-# Java Package Files #
-*.jar
-*.war
-*.ear
-############
+# Code Recommenders
+.recommenders/
 
-### Netbeans ###
-nbproject/private/
-build/
-bin/
-nbbuild/
-dist/
-nbdist/
-nbactions.xml
-nb-configuration.xml
-################
-
-### Linux ###
-.*
-!.gitignore
-!.gitattributes
-!.tx
-!.github/
-*~
-#############
-
-### OpenMRS Files ###
-/webapp/activemq-data
-/webapp/src/main/webapp/WEB-INF/tags/module
-/webapp/src/main/webapp/WEB-INF/dwr-modules.xml
-/webapp/src/main/webapp/WEB-INF/module_messages*
-/web/WEB-INF/dwr-modules.xml
-#############
-
-### log files ###
-**/logs
-*.log
-#############
+# Annotation Processing
+.apt_generated/
+.apt_generated_test/
 
-!.editorconfig
+# Scala IDE specific (Scala & Java development for Eclipse)
+.cache-main
+.scala_dependencies
+.worksheet
 
-### Liquibase snapshot files ###
-liquibase/snapshots/*.xml
-#############
+# Uncomment this line if you wish to ignore the project description file.
+# Typically, this file would be tracked if it contains build/dependency configurations:
+#.project
 
-api/src/main/java/org/openmrs/module/ohricore/scratchpad.txt
+api/src/test/resources/org/openmrs/module/ohricore/include/OHRI-StandardDataset-original.xml
```

Comment:
@smallgod This is not necessary delete the file rather than add it to .gitIgnore or use the IDE Scratches to store any content you may need later

Reply:
@ssmusoke okay done!